lib/pg_ha_migrations/constraint.rb

```diff
@@ -0,0 +1 @@
+PgHaMigrations::CheckConstraint = Struct.new(:name, :definition, :validated)
```

Comment:
I understand why this isn't in the `Relation` file, but is there somewhere else we can put this? A single line file seems strange to my brain, but obviously this isn't a _problem_

Reply:
We could put it right above the constraints method, but...it's just a question of whether we expect to move it later IMO.